mainflux-loadtests/build.sbt

```diff
@@ -0,0 +1,15 @@
+name := "mainflux-loadtests"
+version := "1.0-SNAPSHOT"
+
+enablePlugins(GatlingPlugin)
+
+scalaVersion := "2.12.4"
+
+val gatlingVersion = "2.3.1"
+
+libraryDependencies ++= Seq(
+  "io.gatling.highcharts" % "gatling-charts-highcharts" % gatlingVersion % "test,it",
```

Comment:
Since the whole project is for load testing only, replace `"test,id"` with `Test`.